kedro/framework/cli/cli.py

```diff
@@ -38,6 +39,13 @@
 v{version}
 """
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.ERROR)
+handler = logging.StreamHandler(sys.stderr)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
```

Comment:
Is this not formatted by `kedro` already? Why is this needed?

Reply:
When I added this I had issues with the hook not being called unless this formatting was explicitly on the file, it was not using `kedro`'s. I'm doing some tests to see if I can make it just use `kedro`'s instead.

Reply:
Ok, I recalled why I did this. Since it was not using the config from `kedro` correctly, it was putting the error messages on stdout instead of stderr.